### PR TITLE
Simplify door/lift api

### PR DESCRIPTION
**Pull request by louis1706**
Removing static Method than shouldn't be there

### CODE REVIEW

File: Exiled.API/Features/Lift.cs [hunk 1]
```diff
@@ -264,106 +264,6 @@ public override Quaternion Rotation
...
-        /// <seealso cref="Player.EnableEffect(EffectType, float, bool)"/>
-        public static bool TryMeltPlayer(Player player)
-        {
-            if (player.Position.y is >= 200 or <= -200)
-                return false;
-
-            player.EnableEffect(EffectType.Decontaminating);
-
-            return true;
-        }
```
Comment (louis1706) [inline Exiled.API/Features/Lift.cs:365]: Why is this removed ?
because now when it's Decontamination NW is constantly checking if it's give Decontaminating Effect so it's now useless to have this method

File: Exiled.API/Features/Doors/Door.cs [hunk 2]
```diff
@@ -128,6 +129,15 @@ public bool IsOpen
             set => Base.NetworkTargetState = value;
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the door is close.
+        /// </summary>
+        public bool IsClose
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Doors/Door.cs:135]: Not sure whether it's necessary, however just rename it to `IsClosed` anyway.

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]
What about all the `Lock` methods?

File: Exiled.API/Features/Doors/Door.cs (hunk 2 above)
Comment (louis1706) [inline Exiled.API/Features/Doors/Door.cs:135]:
```suggestion
        public bool IsClosed
```

**Comment by NaoUnderscore** [comment]
Looks fine I guess
